ci-operator/step-registry/azure/deprovision/sp-and-custom-role/azure-deprovision-sp-and-custom-role-commands.sh

```diff
@@ -11,6 +11,11 @@ function run_command() {
     eval "${CMD}"
 }
 
+if [[ "${AZURE_INSTALL_USE_MINIMAL_PERMISSIONS}" == "no" ]]; then
```

Comment:
In `cucushift-installer-rehearse-azure-aks-hypershift-ephemeral-creds-guest` install workflow, `azure-provision-service-principal-hypershift` is called to create an extra sp and save it into `azure_sp_id` without `AZURE_INSTALL_USE_MINIMAL_PERMISSIONS` enabled, it is using `azure-deprovision-sp-and-custom-role` to deprovision the created sp.

Reply:
acked, missed this. 
How about to append the condition in this line https://github.com/openshift/release/pull/61272/files#diff-a10c3a5a25baa7a1299c201470c4279fd0294aa5ae6f08de00ee0566631f6e94L20?
Or no need to introduce the env `AZURE_INSTALL_USE_MINIMAL_PERMISSIONS` in this step?

Reply:
I removed the env AZURE_INSTALL_USE_MINIMAL_PERMISSIONS. 
PTAL again, thanks.

Reply:
Yeah, I prefer to not introduce `AZURE_INSTALL_USE_MINIMAL_PERMISSIONS` env in this step if no others issues.